plugins/linear-genome-view/src/LinearGenomeView/model.ts

```diff
@@ -603,8 +606,14 @@ export function stateModelFactory(pluginManager: PluginManager) {
       /**
        * #action
        */
-      setHighlight(highlight: Required<ParsedLocString> | undefined) {
-        self.highlight = highlight
+      setHighlight(highlight: Required<ParsedLocString>) {
+        self.highlight?.push(highlight)
+      },
+      /**
+       * #action
+       */
+      removeHighlight(highlight: Required<ParsedLocString>) {
+        self.highlight.splice(self.highlight.indexOf(highlight), 1)
```

Comment:
nit: can use`self.highlight.remove(highlight)` as a simplification i believe. it is a special mst method